routers/api/v1/user/user.go

```diff
@@ -70,3 +70,13 @@ func GetInfo(ctx *context.APIContext) {
 	}
 	ctx.JSON(200, &api.User{u.ID, u.Name, u.FullName, u.Email, u.AvatarLink()})
 }
+
+func GetUser(ctx *context.APIContext) {
+	u := ctx.User
+
+	// Hide user e-mail when API caller isn't signed in.
+	if !ctx.IsSigned {
```

Comment:
Since this is returning the info of the signed in user, it probably doesn't need to check IsSigned.